src/shared/fonts_utils.js

```diff
@@ -220,8 +220,8 @@ function readFontIndexData(aStream, aIsByte) {
     throw new FormatError(offsize + ' is not a valid offset size');
   }
 
-  var offsets = [];
-  var i;
+  const offsets = [];
+  let i;
```

Comment:
We should inline this variable in the `for` loops as done above and in other files so we can remove it since it's very local for the loops.